naga/src/front/wgsl/mod.rs

```diff
@@ -20,6 +20,8 @@ pub use crate::front::wgsl::error::ParseError;
 use crate::front::wgsl::lower::Lowerer;
 use crate::Scalar;
 
+pub use crate::front::wgsl::parse::directive::language_extension::ImplementedLanguageExtension as WGSLLanguageExtension;
+
```

Comment:
style: I don't think we need to rename this at all, because Naga != the WebGPU interface, but…

```suggestion
```

style: …if we did, I think it'd be clearer to name this a <code>…Language<strong>Feature</strong></code>.

style: Also, we should use Rust conventions around `UpperCamelCase`, and lowercase all but the first letter of the `WGSL` acronym, i.e., `s/WGSL/Wgsl`.

Combined suggestions:

```suggestion
pub use crate::front::wgsl::parse::directive::language_extension::ImplementedLanguageExtension as WgslLanguageFeature;
```

Reply:
side-note: Spec uses both WGSLLanguageExtension and WGSLLanguageFeatures.